tests/Operation/WatchFunctionalTest.php

```diff
@@ -61,9 +60,7 @@ public function setUp(): void
      */
     public function testGetResumeToken(): void
     {
-        if ($this->isPostBatchResumeTokenSupported()) {
-            $this->markTestSkipped('postBatchResumeToken is supported');
-        }
+        $this->skipIfPostBatchResumeTokenNotSupported();
```

Comment:
This logic is incorrect. The test needs to be skipped if `postBatchResumeToken` _is_ supported (i.e. server version >= 4.0.7).